catalog/nato/stanag4559/catalog-stanag4559-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version. 
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD name="STANAG 4559 Federated Source" id="Stanag4559_Federated_Source"
+         description="STANAG 4559 Federated Source">
+
+        <AD description="The ID of the STANAG 4559 Source" name="STANAG Source ID" id="id"
+            required="true" type="String" default="STANAG4559 Federated Source"/>
+
+        <AD description="The URL of the IOR File to use for the STANAG 4559 CORBA Server"
+            name="Ior File URL" id="iorUrl" required="true" type="String"/>
+
+        <AD description="The Username to be used for authentication with HTTP server."
+            name="HTTP(s) Username" id="cxfUsername"
+            required="false" type="String"/>
+
+        <AD description="The Password to be used for authentication with HTTP server."
+            name="HTTP(s) Password" id="cxfPassword"
+
+            required="false" type="String"/>
+
+        <AD description="The Key to be used in Access Criteria" name="Key" id="key" required="false"
```

Comment:
The key is part of the access criteria, which is actually defined and not used.  The configurable properties ( key, access criteria username and password) will be removed.